walletsetup.go

```diff
@@ -188,7 +188,7 @@ func createWallet(cfg *config) error {
 	}
 
 	fmt.Println("Creating the wallet...")
-	w, err := loader.CreateNewWallet(pubPass, privPass, seed, time.Now())
+	w, err := loader.CreateNewWallet(pubPass, privPass, seed, time.Unix(cfg.Birthday, 0))
```

Comment:
Again, I'm not familiar with `btcwallet` as a standalone wallet. When is this code run vs. the `CreateWallet` RPC call?

Reply:
When being used as a standalone tool, you can create a wallet from the command line with `btcwallet --create`, which will run that code in that snippet